src/components/Lesson/Lesson.js

```diff
@@ -137,4 +138,22 @@ const Lesson = ({ lesson, slug }) => {
   );
 };
 
+Lesson.PropTypes = {
+  lesson: PropTypes.shape({
+    frontmatter: PropTypes.shape({
+      title: PropTypes.string,
+      description: PropTypes.string,
+      timeToCompletion: PropTypes.node,
+      videoLinks: PropTypes.array,
+      readingLinks: PropTypes.array,
+      preReadQuizLink: PropTypes.string,
+      preReadQuiz: PropTypes.any,
+      course: PropTypes.string,
+      defaultTab: PropTypes.string,
+      secondaryExerciseUrl: PropTypes.string
+    })
+  }).isRequired,
+  slug: PropTypes.string
```

Comment:
I believe `slug` is required here, as we use `slug` in this file without checking if it exists.

Reply:
Done.